jobserver/models.py

```diff
@@ -362,17 +362,30 @@ class Project(models.Model):
         related_name="projects",
     )
 
-    name = models.TextField(unique=True)
-    email = models.TextField()
-    project_lead = models.TextField()
-    proposed_start_date = models.DateTimeField()
-    proposed_duration = models.TextField()
-
+    name = models.TextField(blank=True)
+    display_name = models.TextField(blank=True)
+    email = models.TextField(blank=True)
+    project_lead = models.TextField(blank=True)
+    proposed_start_date = models.DateTimeField(null=True, blank=True)
+    proposed_duration = models.TextField(blank=True)
+
+    # store a link to our modified FormA
+    form_url = models.TextField(blank=True)
+
+    # Governance approval
+    governance_approval_notes = models.TextField(blank=True)
```

Comment:
It would perhaps be a slightly premature optimisation as this is an MVP, but just to say I'm pretty sure we'll be wanting more auditability here - who approved, and datetime of approval.

Reply:
It's quite likely this will become a state machine pretty soon, as it may be we'll want to audit _un_ approving too; adding a state for "project commenced"; preventing a transition to `commenced` until approvals have happened; etc. Let's chat about what that might look like soon.